### PR TITLE
Clarify which data is personal data.

### LINKED ISSUE #406: Please copy edit acting on data section
The sentence

> If a [person](https://www.w3.org/TR/privacy-principles/#dfn-individual) could reasonably be identified or re-identified through the combination of [data](https://www.w3.org/TR/privacy-principles/#dfn-data) with other [data](https://www.w3.org/TR/privacy-principles/#dfn-data), then that [data](https://www.w3.org/TR/privacy-principles/#dfn-data) is [personal data](https://www.w3.org/TR/privacy-principles/#dfn-data).

is hard to understand, and in particular it is unclear what the final instance of "data" actually refers to.

### PR DESCRIPTION
**Pull request by jyasskin**
Fixes #406.

@chaals, this doesn't try to fix the larger issues, but I think it handles the particular unclarity you pointed out.

***
<a href="https://pr-preview.s3.amazonaws.com/jyasskin/privacy-principles/pull/410.html" title="Last updated on Mar 13, 2024, 5:14 PM UTC (4e42ea0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/410/a203ae0...jyasskin:4e42ea0.html" title="Last updated on Mar 13, 2024, 5:14 PM UTC (4e42ea0)">Diff</a>

### CODE REVIEW

**Review by darobin — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -2184,7 +2184,7 @@
 </aside>
 
 If a [=person=] could reasonably be identified or re-identified through the combination of [=data=] with other
-[=data=], then that [=data=] is [=personal data=].
+[=data=], then both sets of [=data=] are [=personal data=].
```
Comment (jyasskin) [inline index.html:2187]: From @wseltzer:

```suggestion
[=data=], then each set of [=data=] is [=personal data=].
```

Comment (wseltzer) [reply index.html:2187]: I withdraw this suggestion

**Review by wseltzer — APPROVED** [review]
I withdraw my suggestion and approve the original PR

File: index.html [hunk 2]
```diff
@@ -2184,7 +2184,7 @@
 </aside>
 
 If a [=person=] could reasonably be identified or re-identified through the combination of [=data=] with other
-[=data=], then that [=data=] is [=personal data=].
+[=data=], then each set of [=data=] is [=personal data=].
```
Comment (jyasskin) [inline index.html:2187]: Reverting:
```suggestion
[=data=], then both sets of [=data=] are [=personal data=].
```

**Review by torgo — APPROVED** [review]
lgtm

**Pull request merged**